### ISSUE TITLE
Making Tiles With Zoom 15 Removes Place Names

### ISSUE THREAD

**Issue by Kobusvdwalt** [systemed/tilemaker#711]
Hi there, thanks for this excellent project !

I am trying to generate Zoom15 tiles to get a map that includes names for my small roads.
This works, however when I do this I loose the bigger place names.
Generating with the default config works perfectly fine, but I'd like a little more detail when you zoom in.

Here are some screenshots of the problem:
Zoom 14 Tileset (With Place Names)
<img width="950" alt="Screenshot 2024-04-26 at 09 50 26" src="https://github.com/systemed/tilemaker/assets/3214815/4bc88b83-3e4d-4c9e-b819-4729b733353b">

Zoom 15 Tileset (Without Place Names)
<img width="950" alt="Screenshot 2024-04-26 at 09 50 35" src="https://github.com/systemed/tilemaker/assets/3214815/da910645-300d-4c26-af9f-48c1374be22b">

More Detailed Example of a Town:
Zoom 14 Tileset:
<img width="950" alt="Screenshot 2024-04-26 at 09 49 57" src="https://github.com/systemed/tilemaker/assets/3214815/6f5f3fdd-ea56-4c26-9ead-8be618460bd2">

Zoom 15 Tileset:
<img width="950" alt="Screenshot 2024-04-26 at 09 49 04" src="https://github.com/systemed/tilemaker/assets/3214815/07e028b1-b686-4f30-919c-e8c388d30f4c">

Here is the config I used for the Zoom 15 tiles:
```
{
	"layers": {
		"place":            { "minzoom":  0, "maxzoom": 15 },
		"boundary":         { "minzoom":  0, "maxzoom": 15, "simplify_below": 12, "simplify_level": 0.0003, "simplify_ratio": 2 },

		"poi":              { "minzoom": 12, "maxzoom": 15 },
		"poi_detail":       { "minzoom": 14, "maxzoom": 15, "write_to": "poi"},

		"housenumber":      { "minzoom": 14, "maxzoom": 15 },

		"waterway":         { "minzoom":  8,  "maxzoom": 15, "simplify_below": 12, "simplify_level": 0.0003, "simplify_ratio": 2 },
		"waterway_detail":  { "minzoom": 12,  "maxzoom": 15, "write_to": "waterway" },

		"transportation":             { "minzoom": 4,  "maxzoom": 15, "simplify_below": 13, "simplify_level": 0.0003 },
		"transportation_name":        { "minzoom": 8,  "maxzoom": 15 },

		"building":          { "minzoom": 13, "maxzoom": 15 },

		"water":             { "minzoom": 6,  "maxzoom": 15, "simplify_below": 12, "simplify_level": 0.0003, "simplify_ratio": 2},
		"ocean":             { "minzoom": 0,  "maxzoom": 15, "source": "coastline/water_polygons.shp", "filter_below": 12, "filter_area": 0.5, "simplify_below": 13, "simplify_level": 0.0001, "simplify_ratio": 2, "write_to": "water" },
		"water_name":        { "minzoom": 14, "maxzoom": 15 },
		"water_name_detail": { "minzoom": 14, "maxzoom": 15, "write_to": "water_name" },

		"aeroway":           { "minzoom": 11, "maxzoom": 14 },
		"aerodrome_label":   { "minzoom": 10, "maxzoom": 14 },
		"park":              { "minzoom": 11, "maxzoom": 14 },
		"landuse":           { "minzoom":  4, "maxzoom": 14, "simplify_below": 13, "simplify_level": 0.0003, "simplify_ratio": 2 },
		"urban_areas":       { "minzoom":  4, "maxzoom":  8, "source": "landcover/ne_10m_urban_areas/ne_10m_urban_areas.shp", "source_columns": ["featurecla"], "simplify_below": 7, "simplify_level": 0.0003, "simplify_ratio": 2, "write_to": "landuse" },
		"landcover":         { "minzoom":  0, "maxzoom": 14, "simplify_below": 13, "simplify_level": 0.0003, "simplify_ratio": 2 },
		"ice_shelf":         { "minzoom":  0, "maxzoom":  9, "source": "landcover/ne_10m_antarctic_ice_shelves_polys/ne_10m_antarctic_ice_shelves_polys.shp", "source_columns": ["featurecla"], "simplify_below": 13, "simplify_level": 0.0005, "write_to": "landcover" },
		"glacier":           { "minzoom":  2, "maxzoom":  9, "source": "landcover/ne_10m_glaciated_areas/ne_10m_glaciated_areas.shp", "source_columns": ["featurecla"], "simplify_below": 13, "simplify_level": 0.0005, "write_to": "landcover" },
		"mountain_peak":     { "minzoom": 11, "maxzoom": 15 }
	},
	"settings": {
		"minzoom": 0,
		"maxzoom": 15,
		"basezoom": 15,
		"include_ids": false,
		"combine_below": 15,
		"name": "Tilemaker to OpenMapTiles schema",
		"version": "3.0",
		"description": "Tile config based on OpenMapTiles schema",
		"compress": "gzip",
		"filemetadata": {
			"tilejson": "2.0.0", 
			"scheme": "xyz", 
			"type": "baselayer", 
			"format": "pbf", 
            "tiles": ["https://example.com/liechtenstein/{z}/{x}/{y}.pbf"]
		}
	}
}
```

I read through the lua code, but it was a little hard to follow, I assume we need to update something in there, or make that script work for Z 15, is there a way we could configure that as a variable somewhere ?

**Comment by yhdjyyzk** [comment]
I had a similar problem.

**Comment by systemed** [comment]
#728 should fix this - have a try and let me know if it works for you.

**Comment by yhdjyyzk** [comment]
> #728 should fix this - have a try and let me know if it works for you.

It works, Thanks!

**Comment by systemed** [comment]
Great, thank you. Just merged into master.
